README.md

```diff
@@ -37,12 +37,24 @@ You can install the module from command line:
 $ composer require riskio/idempotency-module
 ```
 
-After installation of the package, you need to complete the following steps:
-
- 1. Enable the module by adding `Riskio\IdempotencyModule` in your `application.config.php` file.
- 2. Copy the `riskio_idempotency.global.php.dist` (you can find this file in the `config` folder of the module) into
+After installation of the package, you have to copy the `riskio_idempotency.global.php.dist` (you can find this file in the `config` folder of the module) into
```

Comment:
Maybe rename the file `riskio_idempotency.global.php` and then keep this sentence, or explicitly describe the renaming process... people might not be as good as you think... :)